diff --git a/tools/gn/bootstrap/bootstrap.py b/tools/gn/bootstrap/bootstrap.py
index 8096364..83f5838 100755
--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -192,7 +192,8 @@
           'ENABLE_LOCATION_SOURCE': 'false',
           'ENABLE_PROFILING': 'false',
           'CAN_UNWIND_WITH_FRAME_POINTERS': 'false',
-          'UNSAFE_DEVELOPER_BUILD': 'false'
+          'UNSAFE_DEVELOPER_BUILD': 'false',
+          'CAN_UNWIND_WITH_CFI_TABLE': 'false',
       })
 
   write_buildflag_header_manually(root_gen_dir,
