--- a/components/machine_intelligence/generic_logistic_regression_inference.cc	2017-10-27 18:05:19.000000000 -0400
+++ b/components/machine_intelligence/generic_logistic_regression_inference.cc.new	2017-10-28 15:07:03.865380457 -0400
@@ -4,6 +4,9 @@
 
 #include "components/machine_intelligence/generic_logistic_regression_inference.h"
 
+#include <string>
+#include <stdio.h>      
+#include <math.h>
 #include "base/logging.h"
 #include "components/machine_intelligence/ranker_example_util.h"
 
